core/daemon/pom.xml

```diff
@@ -14,7 +14,6 @@
   The OpenNMS Daemon launches and acts as a container for OpenNMS Services such as the poller or collector.
   </description>
   <properties>
```

Comment:
We can get rid of the entire properties block here (and in other POMs with similar changes).